src/utilities.ts

```diff
@@ -131,3 +131,11 @@ export const buildStackedBranchPRMessage = (
 
 	return message;
 };
+
+export const wait = (ms: number): Promise<void> => {
```

Comment:
didn't we have this function somewhere already?

Reply:
not sure about that, but i will remove this after i remove that hack with waiting for applied jobs